### ISSUE TITLE
Gitlab API after 11.6.1 use IID for merge_requests

### ISSUE THREAD

**Issue by leslau** [danger/danger#1065]
Gitlab now uses IID instead of ID on API call for merge_requests endpoint

https://docs.gitlab.com/ee/api/README.html#id-vs-iid

https://github.com/danger/danger/blob/ca052e180f8973e0b28b2845a489f27d0996ab1f/lib/danger/ci_source/gitlab_ci.rb#L35

**Comment by orta** [comment]
I think you'll have to figure a way to support both I imagine, because people can still use the older GitLab and we shouldn't break backwards compatability.

**Comment by leslau** [comment]
```ruby
return env["CI_MERGE_REQUEST_IID"] if env["CI_MERGE_REQUEST_IID"]
return env["CI_MERGE_REQUEST_ID"] if env["CI_MERGE_REQUEST_ID"]
```

maybe something like this?

**Comment by orta** [comment]
Yeah, that probably makes sense - I'm not 100% of the context but feels right to me

**Comment by pbendersky** [comment]
This was fixed with #1072
